diff --git a/dist/runtime/Icon.vue b/dist/runtime/Icon.vue
index e68a5d8c5ed7b18f2314a5c76576c71f8dcd2f3c..6ee8026f5f221f469b250fec8faf853fb562a90b 100644
--- a/dist/runtime/Icon.vue
+++ b/dist/runtime/Icon.vue
@@ -8,7 +8,7 @@ import { useAppConfig, useNuxtApp, useState } from '#imports'
 import { resolveIconName } from './utils'
 
 const nuxtApp = useNuxtApp()
-const appConfig = useAppConfig() as {
+const appConfig = useAppConfig() as unknown as {
   nuxtIcon: {
     size?: string
     class?: string
diff --git a/dist/runtime/IconCSS.vue b/dist/runtime/IconCSS.vue
index e54e4e896cc0b0af0f86f6c682b2de20bd773633..a39afcc23e2594646594e3017dc513e27da4d408 100644
--- a/dist/runtime/IconCSS.vue
+++ b/dist/runtime/IconCSS.vue
@@ -3,7 +3,7 @@ import { computed } from 'vue'
 import { useAppConfig } from '#imports'
 import { resolveIconName } from './utils'
 
-const appConfig = useAppConfig() as {
+const appConfig = useAppConfig() as unknown as {
   nuxtIcon: {
     size?: string
     class?: string
